src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -42,21 +43,28 @@ export class LambdaTreeDataProvider implements vscode.TreeDataProvider<AWSTreeNo
         private readonly awsContext: AwsContext,
         private readonly awsContextTrees: AwsContextTreeCollection,
         private readonly regionProvider: RegionProvider,
-        private readonly resourceFetcher: ResourceFetcher
+        private readonly resourceFetcher: ResourceFetcher,
+        private readonly  outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel('AWS Lambda')
     ) {
         this._onDidChangeTreeData = new vscode.EventEmitter<AWSTreeNodeBase | undefined>()
         this.onDidChangeTreeData = this._onDidChangeTreeData.event
         this.regionNodes = new Map<string, RegionNode>()
+        if (!ext.lambdaOutputChannel) {
```

Comment:
Please don't do this piecemeal. It's also out-of-scope for this PR. In this PR, just use `ext.lambdaOutputChannel` instead of the constructor parameter. You can send a separate PR that removes `ext.lambdaOutputChannel` entirely and replaces it with constructor parameters.